loleaflet/src/core/Socket.js

```diff
@@ -1018,6 +1021,21 @@ app.definitions.Socket = L.Class.extend({
 			var mobileEvent = textMsg.substring('mobile: '.length);
 			this._map.fire(mobileEvent);
 		}
+		else if (textMsg.startsWith('blockui: ')) {
+			textMsg = textMsg.substring('blockui: '.length);
```

Comment:
Trim the textMsg, in case it has some extra whitespace, is a good idea.

Reply:
done